### ISSUE TITLE
some `bse` attributes still have ending on them

### ISSUE THREAD

**Issue by jtauber** [gcelano/Thesaurus_Morphologicus_Linguae_Graecae#1]
I wrote a test and found 48 cases in 1st declension file where `bse`+`end` does not equal `key` (even after stripping diacritics):

```
n722 ἄγνοιᾰ
n9781 ἀντιάνειρᾰ
n14905 ἄρδᾰ
n50325 ἱλάειρᾰ
n57561 κίσσᾰ
n58586 κόλλᾰ
n60856 κυνητίνδᾰ
n61945 λαπάρᾱ
n64234 λύσσᾰ
n64828 μάλθᾰ
n64930 μάννᾰ
n65858 μελανθέᾱ
n66048 μέλισσᾰ
n66332 μέρμηρᾰ
n67881 μίνθᾰ
n69923 νάφθᾰ
n71958 Ὄγκᾱ
n73362 ὄλυρᾰ
n75568 ὄσσᾰ
n80021 πάτρᾱ
n80272 πέζᾰ
n80899 πέπειρᾰ
n83266 πίσσᾰ
n86279 πρέσβᾰ
n86280 πρέσβεᾰ
n91213 πτοίᾱ
n91826 πύρρᾱ
n19696 βδέλλᾰ
n92255 Ῥέᾱ
n95308 Σκύλλᾰ
n101560 σφαῖρᾰ
n101754 σφῦρᾰ
n104224 τόλμᾰ
n104376 τόρμᾰ
n105351 τριττύᾱ
n106120 ὕαινᾰ
n106205 ὑγίειᾰ
n111930 φοίνᾱ
n21500 γαλήνειᾰ
n25401 διάνοιᾰ
n28002 δόξᾰ
n28507 δρύππᾱ
n38903 ἐπίβδᾰ
n44249 εὔκλειᾰ
n44552 εὔνοιᾰ
n45023 εὐρυόδειᾰ
n46169 ζεῦγλᾰ
n47958 θέαινᾰ
```

On a potentially related note: is there any reason you retain the accent on `end` but strip it from `bse`?

**Comment by gcelano** [comment]
Hi James,

Thank you very much for your hint! Yes, they were mistake I have now been
able to correct, as you can see online. Sorry if I did not answer your last
email before, but I have been recently particularly busy.

I would like to extract as much morphological information as possible from
the LSJ. This is why I would like to extract declensions and verbs. I see
that you have already done a lot and I would really like to join forces.
Can I ask you where/how you created your lexica (especially the one
containing the verbs with their tenses)?

My work is a little bit slow right now because I am about to publish
treebanked data, and this is taking some time. As for your question
end/bse, I retain the accent on the endings because, in general, when
segmenting a word, I would like to preserve distinctions among endings also
on the basis of the accent, i.e., I think that this should be in general
much more feasible than dealing with the accent on the base. Originally I
wanted to write rules for creating declined nouns with accents, but then I
thought that it is maybe best to start without taking into consideration
the accent on the base.

What are you projects?

Giuseppe



2015-07-04 1:12 GMT+02:00 James Tauber <notifications@github.com>:

> I wrote a test and found 48 cases in 1st declension file where bse+end
> does not equal key (even after stripping diacritics):
>
> n722 ἄγνοιᾰ
> n9781 ἀντιάνειρᾰ
> n14905 ἄρδᾰ
> n50325 ἱλάειρᾰ
> n57561 κίσσᾰ
> n58586 κόλλᾰ
> n60856 κυνητίνδᾰ
> n61945 λαπάρᾱ
> n64234 λύσσᾰ
> n64828 μάλθᾰ
> n64930 μάννᾰ
> n65858 μελανθέᾱ
> n66048 μέλισσᾰ
> n66332 μέρμηρᾰ
> n67881 μίνθᾰ
> n69923 νάφθᾰ
> n71958 Ὄγκᾱ
> n73362 ὄλυρᾰ
> n75568 ὄσσᾰ
> n80021 πάτρᾱ
> n80272 πέζᾰ
> n80899 πέπειρᾰ
> n83266 πίσσᾰ
> n86279 πρέσβᾰ
> n86280 πρέσβεᾰ
> n91213 πτοίᾱ
> n91826 πύρρᾱ
> n19696 βδέλλᾰ
> n92255 Ῥέᾱ
> n95308 Σκύλλᾰ
> n101560 σφαῖρᾰ
> n101754 σφῦρᾰ
> n104224 τόλμᾰ
> n104376 τόρμᾰ
> n105351 τριττύᾱ
> n106120 ὕαινᾰ
> n106205 ὑγίειᾰ
> n111930 φοίνᾱ
> n21500 γαλήνειᾰ
> n25401 διάνοιᾰ
> n28002 δόξᾰ
> n28507 δρύππᾱ
> n38903 ἐπίβδᾰ
> n44249 εὔκλειᾰ
> n44552 εὔνοιᾰ
> n45023 εὐρυόδειᾰ
> n46169 ζεῦγλᾰ
> n47958 θέαινᾰ
>
> On a potentially related note: is there any reason you retain the accent
> on end but strip it from bse?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gcelano/Thesaurus_Morphologicus_Linguae_Graecae/issues/1>
> .
>



-- 
Dr. Giuseppe G. A. Celano
Postdoctoral Researcher
Universität Leipzig
Institute of Computer Science,
​Section of ​
Digital
​ ​
Humanities
​

Augustusplatz 10
04109 Leipzig
​ (DE)​

E-mail: giuseppegacelano@gmail.com
            giuseppe.celano@tufts.edu
            celano@informatik.uni-leipzig.de
Web site 1: http://www.dh.uni-leipzig.de/wo/team/
Web site 2: https://sites.google.com/site/giuseppegacelano/

**Comment by gcelano** [comment]
Thanks for this. I did not consider in my script the possibility for @key to contain words with final vowels marked with a brevis or macron. Now I fixed the script. I retain accent on end(ing) because I want to distinguish (at least) endings also on the basis of accent.

**Comment by jtauber** [comment]
fixed by 62719eeeed4b1bc76f21a5e8187e3b2d5dcb06eb
